ros2param/test/test_verb_dump.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import threading
+
+# import pytest
+import unittest
+
+import rclpy
+
+from rclpy.executors import MultiThreadedExecutor
+from ros2cli import cli
+
+TEST_NODE = 'test_node'
+TEST_NAMESPACE = ''
+
+
+class TestVerbDump(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.context = rclpy.context.Context()
+        rclpy.init(context=cls.context)
+        cls.node = rclpy.create_node(
+            TEST_NODE, namespace=TEST_NAMESPACE, context=cls.context)
+
+        cls.executor = MultiThreadedExecutor(context=cls.context, num_threads=2)
+        cls.executor.add_node(cls.node)
+
+        cls.node.declare_parameter('bool_param', True)
+        cls.node.declare_parameter('int_param', 42)
+        cls.node.declare_parameter('double_param', 1.23)
+        cls.node.declare_parameter('str_param', 'Hello World')
+        cls.node.declare_parameter('foo/str_param', 'foo')
```

Comment:
I'm not sure if namespaces are delimited with `/` or `.` as I have seen both :s .